packages/auto_mappr/README.md

```diff
@@ -488,6 +489,37 @@ This is by design and it is up to developer decide if bang operator is appropiat
 
 Precedense is `global configuration` -> `MapType` -> `Field` -> defaults to false.
 
+### Safe mapping
+Sometimes it can happen that mapping goes wrong and exception is thrown. Typically it can be caused 
+by mapping from source with a nullable variable to a target with a corresponding non-nullable variable. 
+If you want to avoid propagating the exception through the app, auto_mappr can catch it and return null instead. In case you want to know
+that something went wrong during the mapping there is a `onMappingError` callback that is invoked each time error occurs. If you want enable
+safe mapping, it can be done either on the global level in `build.yaml` with `safeMapping` option
+
+  ```yaml
+   $default:
+    builders:
+      auto_mappr:
+        options:
+          safeMapping: true
+  ```
+
+or individually for each `MapType`:
+
+```dart
+@AutoMappr([
+  MapType<UserDto, User>(
+    safeMapping: true,
+  ),
+])
+class Mappr extends $Mappr {}
+```
+Note that global `safeMapping` has always the priority.
```

Comment:
Maybe explain what this means in more detail. What happends when I set `safeMapping: false` locally, while I have global option turned on?

Reply:
Precedense of safe mapping is defined.